pyuvdata/parameter.py

```diff
@@ -103,34 +104,47 @@ def __eq__(self, other):
             # only check that value is identical
             if not isinstance(self.value, other.value.__class__):
                 print(
-                    "{name} parameter value classes are different. Left is "
-                    "{lclass}, right is {rclass}".format(
-                        name=self.name,
-                        lclass=self.value.__class__,
-                        rclass=other.value.__class__,
-                    )
+                    f"{self.name} parameter value classes are different. Left is "
+                    f"{self.value.__class__}, right is {self.value.__class__}"
```

Comment:
I think this should be
```suggestion
                    f"{self.value.__class__}, right is {other.value.__class__}"
```

Reply:
yes, thanks for catching that!